web/templates/user_profile_modal.hbs

```diff
@@ -62,7 +62,7 @@
                                 </div>
                             </div>
                             <div class="col-wrap col-right">
-                                <div id="avatar" {{#if user_is_guest}} class="guest-avatar" {{/if}}
+                                <div id="avatar" class="{{#if user_is_guest}}guest-avatar{{else}}{{#if is_bot}}bot-avatar{{/if}}{{/if}}"
```

Comment:
This is not consistent with how we have these classes at other places and will lead to different avatars for "guest bots" at different places. I think we should show the color used for bots only for "guest bot" users, but I am not sure.

Depending on what color do we want to show for "guest bots", there can be two ways to implement this - either we can keep both the classes "guest-avatar" and "bot-avatar" and make the CSS handle what to show or we can only have one class, since these classes are only used to set the bottom triangle properties and nothing else.

In any case, we should make it consistent across all places where we show avatar.

And just as a reference for future use, you can directly use `{{else if is_bot}}` and do not need to have `{{else}}{{#if is_bot}}`.

Reply:
Done.
We used one class and use one colour for the bot users.

Reply:
Agreed that consistency is an important thing to achieve.

Reply:
I think we should keep the triangle color same for all bots, whether they are guests or not.

Reply:
@sahil839 done